Fleece/PlatformCompat.hh

```diff
@@ -57,6 +63,12 @@
 
     #define StackArray(NAME, TYPE, SIZE)    TYPE NAME[(SIZE)]
 
-    #define MKDIR(PATH, MODE) ::mkdir(PATH, (mode_t)MODE)
+    #define mkdir_u8(PATH, MODE) ::mkdir(PATH, (mode_t)MODE)
```

Comment:
...and if this stuff is going into a namespace, these macros should be changed into inline functions so they can be called with `fleece::`.